.config/.env.1password

```
@@ -1,4 +1,4 @@
 export DX_ANTHROPIC_API_KEY="op://CI Vault/hub.dxos.network/shared/ANTHROPIC_API_KEY"
+export CLOUDFLARE_API_TOKEN="op://Shared/script runtime uploader token/credential"
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CLOUDFLARE_API_TOKEN key should go before the DX_ANTHROPIC_API_KEY key

(UnorderedKey)